src/state/contexts/CustomersContext.js

```diff
@@ -25,6 +26,8 @@ const CustomersProvider = ({ children }) => {
       value={{
         custInfo,
         setCustInfo,
+        // pets,
```

Comment:
why are these lines commented out?

Reply:
This was part of the previous test and has also been removed from CustomersContext. It resides in PetsContext now.